stories/webapp/accountdashboard/WordPressIntegrationStory.js

```diff
@@ -0,0 +1,105 @@
+import React, { Fragment } from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { withModalTrigger, Modal, Button, InputWithButton, Icon } from 'lib';
+
+const stories = storiesOf('Web app', module);
+
+const ModalTrigger = withModalTrigger({
+  children: <Icon name="integration" />,
+  types: ['icon-only']
+});
+
+stories.add('WordPressIntegrationStory', () => {
+  const modalSubHeader = (
+    <Fragment>
+      <Icon name="integration" className="m-quarter mr-half" />
+      Wordpress v5.4
+      <span className="text-xs text-green-primary bg-green-90 px-quarter py-tenth inline-flex rounded-sm font-semi-bold mx-quarter">
+        CONNECTED
+      </span>
```

Comment:
🚩 We should have a component called Pill that basically does this styling for you

Reply:
Yes - unfortunately we _do_ have a component called `Pill` and it does do styling, but I needed to add a bunch of styles to override the styling. There's also a class `.pill` in the app - which we use for the purple BETA labels. I didn't want to clash with that, but if you think it's worth it I'll split this into a new PR to update our gather-ui Pill component - offering size/colour variants.